x/leverage/keeper/liquidate.go

```diff
@@ -210,15 +216,24 @@ func ComputeCloseFactor(
 		return sdk.OneDec()
 	}
 
-	// outside of special cases, close factor scales linearly between MinimumCloseFactor and 1.0,
-	// reaching max value when (borrowed / threshold) = 1 + CompleteLiquidationThreshold
+	if collateralValue.LTE(liquidationThreshold) {
```

Comment:
I think this should not be possible. Liquidation threshold is computed based on collateral value, which is basically

    sum(t \in collateral, t * usd_rate(t) * t.liquidation_threshold)

For each token, it's liquidation threshold must be strictly < 1. We need to add that check in params validation.
We can keep the check here, but we should add a comment that normally this is not possible.

Reply:
Correct, it should be impossible - I'll add a comment, but the check here (especially for equality) is done to protect the next calculation (especially from panic on division by zero).

Reply:
From `Token.Validate()`

```
// Liquidation threshold ranges between collateral weight and 1, inclusive.
if t.LiquidationThreshold.LT(t.CollateralWeight) || t.LiquidationThreshold.GT(sdk.OneDec()) {
	return fmt.Errorf("invalid liquidation threshold: %s", t.LiquidationThreshold)
}
```

I'll change that `GT` to `GTE` in in this PR.